<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>921</width>
    <height>605</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QWidget" name="widget_2" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>0</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QTextEdit" name="textEdit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>150</height>
          </size>
         </property>
         <property name="html">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Microsoft YaHei UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;CL:(HHV(H,SHORT)+LLV(L,SHORT))/2,COLORGRAY;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;DL:(HHV(H,MID)+LLV(L,MID))/2,COLORMAGENTA;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;LL:REFX(C,MID),COLORBLUE;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;A:REF((CL+DL)/2,MID),COLORRED,DOTLINE;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;B:REF((LLV(L,LONG)+HHV(H,LONG))/2,MID),COLORGREEN,DOTLINE;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;STICKLINE(A&amp;lt;B,A,B,0.05,-1),COLOR449944;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;STICKLINE(A&amp;gt;=B,A,B,0.05,-1),COLORFF0000;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>50</height>
          </size>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QTableWidget" name="tableWidget">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <row>
             <property name="text">
              <string/>
             </property>
            </row>
            <row>
             <property name="text">
              <string/>
             </property>
            </row>
            <row>
             <property name="text">
              <string/>
             </property>
            </row>
            <row>
             <property name="text">
              <string/>
             </property>
            </row>
            <row>
             <property name="text">
              <string/>
             </property>
            </row>
            <row>
             <property name="text">
              <string/>
             </property>
            </row>
            <row>
             <property name="text">
              <string/>
             </property>
            </row>
            <row>
             <property name="text">
              <string/>
             </property>
            </row>
            <row>
             <property name="text">
              <string/>
             </property>
            </row>
            <row>
             <property name="text">
              <string/>
             </property>
            </row>
            <row>
             <property name="text">
              <string/>
             </property>
            </row>
            <column>
             <property name="text">
              <string>参数名</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>值</string>
             </property>
            </column>
            <item row="0" column="0">
             <property name="text">
              <string>LONG</string>
             </property>
            </item>
            <item row="0" column="1">
             <property name="text">
              <string>4</string>
             </property>
            </item>
            <item row="1" column="0">
             <property name="text">
              <string>MID</string>
             </property>
            </item>
            <item row="1" column="1">
             <property name="text">
              <string>3</string>
             </property>
            </item>
            <item row="2" column="0">
             <property name="text">
              <string>SHORT</string>
             </property>
            </item>
            <item row="2" column="1">
             <property name="text">
              <string>2</string>
             </property>
            </item>
            <item row="3" column="0">
             <property name="text">
              <string>P1</string>
             </property>
            </item>
            <item row="3" column="1">
             <property name="text">
              <string>6</string>
             </property>
            </item>
            <item row="4" column="0">
             <property name="text">
              <string>P2</string>
             </property>
            </item>
            <item row="4" column="1">
             <property name="text">
              <string>12</string>
             </property>
            </item>
            <item row="5" column="0">
             <property name="text">
              <string>P3</string>
             </property>
            </item>
            <item row="5" column="1">
             <property name="text">
              <string>24</string>
             </property>
            </item>
            <item row="6" column="0">
             <property name="text">
              <string>M1</string>
             </property>
            </item>
            <item row="6" column="1">
             <property name="text">
              <string>6</string>
             </property>
            </item>
            <item row="7" column="0">
             <property name="text">
              <string>M2</string>
             </property>
            </item>
            <item row="7" column="1">
             <property name="text">
              <string>12</string>
             </property>
            </item>
            <item row="8" column="0">
             <property name="text">
              <string>M3</string>
             </property>
            </item>
            <item row="8" column="1">
             <property name="text">
              <string>24</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QRadioButton" name="radioMain">
              <property name="layoutDirection">
               <enum>Qt::LeftToRight</enum>
              </property>
              <property name="text">
               <string>主图</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioSub">
              <property name="layoutDirection">
               <enum>Qt::LeftToRight</enum>
              </property>
              <property name="text">
               <string>副图</string>
              </property>
              <property name="checked">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_err">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnAdd">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>add</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnClear">
              <property name="text">
               <string>clear</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_4" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>0</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="KChartView" name="kchart0" native="true"/>
       </item>
       <item>
        <widget class="KChartView" name="kchart1" native="true"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KChartView</class>
   <extends>QWidget</extends>
   <header location="global">src/kchartview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
